catalogue_api/api/src/test/scala/uk/ac/wellcome/platform/api/ApiSwaggerTest.scala

```diff
@@ -117,7 +118,7 @@ class ApiSwaggerTest extends FunSpec with Matchers with fixtures.Server {
       "es.index.v2" -> "v2"
     )
 
-    val response: Response = withServer(flags) { server =>
+    val response: Response = withServer(new Server,flags) { server =>
```

Comment:
I know the API doesn’t have default flags, but I’d have preferred to keep the withServer fixture signature the same throughout – like you’ve done with the ingestor and ID minter.